tests/integration/high_availability/high_availability_helpers.py

```diff
@@ -228,6 +239,45 @@ async def relate_mysql_and_application(
     )
 
 
+def deploy_chaos_mesh(ops_test: OpsTest, namespace: str) -> None:
```

Comment:
Doesn't look like ops_test is used

Reply:
Good catch! Addressed in be15a67bd659c981a1705d287ef1d10a9e930f38